plugin/src/main/kotlin/com/github/triplet/gradle/play/ProcessPackageMetadataTask.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.triplet.gradle.play
+
+import com.android.build.gradle.api.ApkVariantOutput
+import com.github.triplet.gradle.play.internal.PlayPublishTaskBase
+import com.github.triplet.gradle.play.internal.ResolutionStrategy
+import org.gradle.api.tasks.TaskAction
+
+open class ProcessPackageMetadataTask : PlayPublishTaskBase() {
+    @TaskAction
+    fun process() {
+        if (extension._resolutionStrategy != ResolutionStrategy.IGNORE) processVersionCodes()
```

Comment:
Let's only process version codes if strategy is AUTO. The FAIL case in `processVersionCodes()` is a nice shortcut but I think it's not worth the extra server roundtrip just to fail a little faster. Remember, we would introduce this check (and fail the build) even if the developer did not even intent to publish the apk